# This message contains an array of events
std_msgs/Header header
uint32 height
uint32 width
Event[] events

